CharacterCreation/05_ArmourList.txt

```diff
@@ -656,9 +651,9 @@ Secondary Module Benefit: -12 Nanite Activation Cost,
 UTILITY HARD POINT EQUIPMENT
 ==============================
 
-	=====   LEVEL 1 REQUIREMENT   =====
+=====   LEVEL 1 REQUIREMENT   =====
 	
-	---- Explosives Carrier ----
+---- Explosives Carrier ----
 Primary Module Cost:	375
 Explosives Skill Req:	10
 Prim Module Hard Pts: 	1

```

Comment:
I think it would work best for these description paragraphs would be to remove the "description:" and leave them as paragraphs. I think that it's pretty self-evident that these are descriptions. Also, they'll parse better.

Reply:
What are we going to do about the rest of the items that use that terminology? Notably, every weapon with additional information uses something like "Description/Notes:"

Also, removing the item denotation ruins the consistency of the system; however, since the primary goal is to get CXDocs up and running, we can probably remove all the Description/Notes text from every item file.

Reply:
Let's hold off on that work until I can talk through that with ya. There might be a better long term solution.

Reply:
alrighty